ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -53,11 +54,11 @@ import com.daml.ledger.api.{domain => LedgerApiDomain}
 import com.daml.ports.Port
 
 object AbstractHttpServiceIntegrationTestFuns {
-  private[http] val dar1 = requiredResource("docs/quickstart-model.dar")
+  private[http] val dar1 = requiredResource(ModelTestDar.path)
```

Comment:
I'm not sure about this change either.

Reply:
The context for this change is that the `"docs/quickstart-model.dal"` defines a template with module and entity name `Iou` similar to one in `ModelTestDar` and due to that fact that we were keeping the ledger around this was causing the `Template(None, Iou, Iou)` to drop out of the unique map on `TemplateIdMap(below)` 

```scala
  case class TemplateIdMap(
      all: Set[TemplateId.RequiredPkg],
      unique: Map[TemplateId.NoPkg, TemplateId.RequiredPkg],
  )
```  

and thus resulting in a test failures on not being able to resolve template